### PR TITLE
636 - add width and hight to the media infos

### PR DESCRIPTION
**Pull request by flaviaF**
https://github.com/backbee/backbee-standard/issues/636

### CODE REVIEW

File: Rest/Controller/ResourceController.php [hunk 1]
```diff
@@ -94,6 +94,12 @@ private function doRequestUpload(UploadedFile $file)
                 if ($file->getClientSize() <= $file->getMaxFilesize()) {
                     $data = $this->buildData($file->getClientOriginalName(), $file->guessExtension());
                     $file->move($tmpDirectory, $data['filename']);
+                    $data['size'] = round($file->getClientSize() / 1024, 2);
```
Comment (eric-chau) [inline Rest/Controller/ResourceController.php]: Why do you divide it by 1024? To get the size in Kb unit?

File: Rest/Controller/ResourceController.php [hunk 2]
```diff
@@ -94,6 +94,12 @@ private function doRequestUpload(UploadedFile $file)
...
+                    $imageInfo = getimagesize( $data['path']);
+                    if(isset($imageInfo[0]) && isset($imageInfo[1])) {
```
Comment (eric-chau) [inline Rest/Controller/ResourceController.php]: Add a space between `if` and `(` please =)

**Comment by eric-chau** [comment]
Good job @flaviaF =) :+1:

File: Rest/Controller/ResourceController.php (hunk 1 above)
Comment (flaviaF) [inline Rest/Controller/ResourceController.php]: Yes :)

File: Rest/Controller/ResourceController.php [hunk 3]
```diff
@@ -94,6 +94,12 @@ private function doRequestUpload(UploadedFile $file)
                 if ($file->getClientSize() <= $file->getMaxFilesize()) {
                     $data = $this->buildData($file->getClientOriginalName(), $file->guessExtension());
                     $file->move($tmpDirectory, $data['filename']);
+                    $data['size'] = round($file->getClientSize() / 1024, 2);
+                    $imageInfo = getimagesize( $data['path']);
```
Comment (crouillon) [inline Rest/Controller/ResourceController.php]: Beware getimagesize() can trigger an error, $imageInfo = @getimagesize( $data['path']);

On the other hand, when an error occured, may be $data['height'] and $data['width'] should be set to 0

**Comment by crouillon** [comment]
Thx @flaviaF , :+1:

**Pull request merged**